### ISSUE TITLE
sandbox should ignore unnecessary files as much as possible

### ISSUE THREAD

**Issue by exalted** [architect/architect#1213]
Sometimes sandbox will crash with the following error:

```
× Error: EMFILE: too many open files, watch
    at FSEvent.FSWatcher._handle.onchange (node:internal/fs/watchers:204:21)
```

I believe this is due to too many files matched to be watched [here](https://github.com/architect/sandbox/blob/e3aa6e0001c4c54296c26eba563e4fe6f8e05417/src/cli/index.js#L28).

Apparently `node-watch` has filter/skip option and at the very least we could "ignore" node_modules in order to drastically reduce the number of files to be watched.

Initially discussed on Slack here: https://architecture-as-text.slack.com/archives/CMK98634P/p1630664011009800

**Comment by exalted** [comment]
Something like this, taken straight from node-watch's readme:

```
watch('./', {
  recursive: true,
  filter(f, skip) {
    // skip node_modules
    if (/\/node_modules/.test(f)) return skip;
    // skip .git folder
    if (/\.git/.test(f)) return skip;
    // only watch for js files
    return /\.js$/.test(f);
  }
});
```

**Comment by ryanblock** [comment]
Fix is available for testing in Arc 9.0.1 RC (`npm i @architect/architect@RC`)!

**Comment by exalted** [comment]
Thanks @ryanblock . https://github.com/architect/sandbox/pull/642/commits/457572bf4777aa93c2509917ff68fad52792b5e0#diff-f00e7c78fa83a1316c2844a481cc2eeda7c7c2c4f958d39e9e457ddbc6b5c20f looks good to me.

**Comment by ryanblock** [comment]
Fixed in Arc 9.0.1!
